### PR TITLE
Fixes #8374: Error in cf-serverd for allowing remote access to cf-agent binary

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/8374

### CODE REVIEW

File: initial-promises/node-server/common/1.0/cf-served.cf [hunk 1]
```diff
@@ -58,11 +58,17 @@ bundle server access_rules
         maproot => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  },
         admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  };
 
-    any::
-      # Allow server to remotely run the agent
-      "${sys.cf_agent}"
+    # Allow server to remotely run the agent
+    # Using ${sys.cf_agent} fails, as cf-serverd cannot canonize its path
+    policy_server_set.linux::
```
Comment (jooooooon) [inline initial-promises/node-server/common/1.0/cf-served.cf]: Why are you introducing the policy_server_set class here?

Comment (ncharles) [inline initial-promises/node-server/common/1.0/cf-served.cf]: to have a synchronized config between initial promises and normal techniques

Comment (ncharles) [inline initial-promises/node-server/common/1.0/cf-served.cf]: and prevent a random rudder machine to connect to a non configured instance

**Comment by jooooooon** [comment]
OK.

**Pull request merged**